filament/CMakeLists.txt

```diff
@@ -523,6 +523,9 @@ add_library(${TARGET} STATIC ${PRIVATE_HDRS} ${PUBLIC_HDRS} ${SRCS} ${DATA_BINS}
 # specify where the public headers of this library are
 target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 
+# add this subproject to the Core folder
+set_target_properties(${TARGET} PROPERTIES FOLDER Core)
```

Comment:
What about we use "Filament" instead of "Core", so it mimics better the folder names?

Reply:
Makes sense; should “shaders” and “vkShaders” go into a separate sub-folder?  They’re currently in the Core/Filament folder, but since they appear to be auto-generated from the .vs and .fs files, I’m not sure if they should be on the same categorical level…

Reply:
Any generated code should in a sub-folder called Generated if possible. It's stuff a developer is not supposed to look at (unless working on the code generator itself).

Reply:
Alrighty; these are now in the Filament/Generated subfolder